src/librustc/session/config.rs

```diff
@@ -112,6 +112,23 @@ pub enum OutputType {
     DepInfo,
 }
 
+/// The epoch of the compiler (RFC 2052)
+///
+/// Pronounced e-poach
+#[derive(Clone, Copy, Hash, PartialOrd, Ord, Eq, PartialEq)]
+pub enum Epoch {
+    /// The 2015 epoch
+    Epoch2015 = 2015,
+    /// The 2018 epoch
+    Epoch2018 = 2018,
+
```

Comment:
should this enum be non-exhasitve?

that's a little, uh, existential of a question I guess...

Reply:
Yeah, this makes sense.